worlds/LauncherComponents.py

```diff
@@ -103,6 +103,8 @@ def launch_textclient():
     Component('FF1 Client', 'FF1Client'),
     # Pokémon
     Component('Pokemon Client', 'PokemonClient', file_identifier=SuffixIdentifier('.apred', '.apblue')),
+    # Pokémon Emerald
+    Component('Pokemon Emerald Client', 'PokemonEmeraldClient', file_identifier=SuffixIdentifier('.apemerald')),
```

Comment:
This isn't needed. You already do this in your world.

Reply:
Fixed in acb7bdc